WooCommerce/WooCommerceTests/ViewRelated/Orders/Issue Refund/RefundItemViewModelTests.swift

```diff
@@ -0,0 +1,43 @@
+import XCTest
+import Yosemite
+
+@testable import WooCommerce
+
+/// Test cases for `RefundItemViewModel`
+///
+final class RefundItemViewModelTests: XCTestCase {
+    func test_viewModel_is_created_with_correct_initial_values() {
+        // Given
+        let item = sampleItem()
+        let product = sampleProduct()
+        let currencySettings = CurrencySettings()
+
+        // Wgeb
+        let viewModel = RefundItemViewModel(item: item, product: product, currency: "usd", currencySettings: currencySettings)
+
+        // Then
+        XCTAssertEqual(viewModel.productImage, product.images.first?.src)
+        XCTAssertEqual(viewModel.productTitle, item.name)
+        XCTAssertEqual(viewModel.quantityToRefund, "0")
+
+        let localizedFormat = NSLocalizedString("%@ x $%@ each", comment: "")
+        XCTAssertEqual(viewModel.productQuantityAndPrice, String(format: localizedFormat, "\(item.quantity)", item.price))
```

Comment:
I think we might want to avoid creating a different localizable string in unit tests? In `RefundItemViewModel`, it is using `NSLocalizedString("%@ x %@ each", comment: ...)` without the currency symbol assumed in the test `$`.

Alternatively, we can keep the localizable string as in the view model and then update the next assert to be:

```suggestion
        let localizedFormat = NSLocalizedString("%@ x %@ each", comment: "")
        XCTAssertEqual(viewModel.productQuantityAndPrice, String(format: localizedFormat, "\(item.quantity)", "$\(item.price)"))
```

Reply:
Nice catch!